examples/rl/pendulum/ex_save_and_load.py

```diff
@@ -0,0 +1,157 @@
+import functools
+import pickle
+import sys
+import time
+from datetime import datetime
+
+# Note: mujoco viewer must load before jax
+# isort: off
+import mujoco
```

Comment:
Consider importing `mujoco.viewer` in the `__init__.py` file right after we import `mujoco` to resolve this error just like #38 - I think it would work.

Reply:
I like this idea, but wasn't able to get it to work. In that case if `jax` gets loaded before anything `ambersim` - which happens in like all of our tests - then the error happens. 

BTW is this also an issue for you? Or is it just something weird about my setup?

Reply:
Actually I just tried pulling your fork and running it while importing the viewer after jax and it seemed to work fine in `test` mode (i imported the viewer first during training, but this shouldn't matter)

Reply:
So I went on quite an adventure to try to figure out what was going on here, but I think I have a fix in #44. We shouldn't need these import order hacks once that lands.

Reply:
Agreed.